src/components/Menu.vue

```diff
@@ -134,7 +135,22 @@
               v-if="session.voteHistory.length"
               @click="toggleModal('voteHistory')"
             >
-              Nomination history<em>[V]</em>
+              Vote history<em>[V]</em>
+            </li>
+            <li @click="toggleRecordVoteHistory" v-if="!session.isSpectator">
+              Permit vote history
```

Comment:
There are 3 entries for the Vote History on the session menu now, I think that's 2 too many. Here's an idea:
always show "Vote History" link in the Storyteller session menu (only show it for players when allowed and non-empty), and keep the "delete" and "disable" options as control elements for the ST *inside* the vote history modal.
You might have to add an extra line below the headline for this to work visually, or restrict the number of entries on the modal (or add pagination there).